### ISSUE TITLE
bfgminer crashes with more than 13 Dualminers

### ISSUE THREAD

**Issue by cbeech** [luke-jr/bfgminer#556]
I have 93 dualminers attached to a raspberry pi running wheezy. I have compiled the head of the master with --enable-scrypt and ran bfgminer -S noauto -S dualminer:all --set-device dualminer:clock=850 --scrypt. 13 dualminers are initialised and their leds are turned on, the remaining dualminers are uninitialised and the raspberry pi crashes becoming unresponsive.

I have been able to use 93 dualminers with https://github.com/dualminer/dualminer-cgminer but would like to upgrade to bfgminer if it is possible to support 93 dualminers.

**Comment by luke-jr** [comment]
Does BFGMiner crash, or the RPi?

**Comment by cbeech** [comment]
The RPi crashes entirely when running bfgminer.

**Comment by nwoolls** [comment]
Doubt I can address this, I have a single DualMiner.

@luke-jr didn't you have some settings that supposedly (or may) allow more serial devices to run on a Pi?

**Comment by luke-jr** [comment]
Yes, but not sure that's going to help if it's crashing. If it didn't make it into docs, though, I'm not sure where I'd have saved that stuff. :(

**Comment by cbeech** [comment]
If there is a way of increasing the serial devices on a Pi then I will give it a try to see if it makes a difference.

**Comment by ghost** [comment]
I'm seeing what I think is the same problem, but with 29 DualMiners. 

I built BFGMiner today from the latest sources. When I run it, the Pi completely crashes. 

Since I've seen problems in the past with attempting to access GridSeed devices from an alternate miner after cgminer has run, I disabled my cgminer service and rebooted. But when I fired up BFGMiner I got the same Pi crash.

So now I'm back using cgminer, which is so old I can't mine on certain pools.

Any further thoughts on how I can get this to work or help diagnose the problem?

**Comment by luke-jr** [comment]
Maybe try starting with just 1 and adding them until it crashes? See if it's a gradual "degrade" or if it goes straight to crashing at some #
